common/postgres-init.sql

```diff
@@ -0,0 +1,4 @@
+CREATE ROLE dbuser WITH LOGIN ENCRYPTED PASSWORD 'dbpwd';
```

Comment:
Same comment here as with the open api file. DB and DB access is exclusive to server / service and not the client, or any other non-server external entity. Please move this to server as well.